src/types.ts

```diff
@@ -111,6 +111,7 @@ export interface ClientConfig {
   headers?: Record<string, string>
 
   ignoreBrowserTokenWarning?: boolean
+  ignoreWarnings?: string | RegExp | Array<string | RegExp>
```

Comment:
Let's add some documentation here (or somewhere else)? Not sure why we don't have it for the other ones…

Especially the behavior that a `string` does a substring match (and not an exact match) is quite crucial.

Reply:
Totally!